packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -2527,11 +2525,9 @@ describe('network stubbing', function () {
         })
       })
 
-      it('when body contains ascii', function (done) {
+      it('when body contains ascii', function () {
         cy.intercept('/post-only', function (req) {
           expect(req.headers['content-length']).to.eq('18')
-
-          done()
```

Comment:
Fixes https://github.com/cypress-io/cypress/issues/23101. We didn't need `done()` here -- passing and failing test cases work as expected here without calling `done()`